r4e-image.sh

```diff
@@ -1,18 +1,27 @@
+#!/bin/bash
 # script for creating an image that points to a certain ostree commit from web server
 # the image includes the agent RPM that communicates with server that monitors edge devices
 # at the end the following products will be available for download:
 #   - ostree image
 #   - kickstart file
 # input:
-#  - IP or hostname with port if different than 80 used for connecting to the local web server
-#  - a name for the image. allows using different ostree image for different types of edge devices
-#  - URL for connecting to the server that monitors edge devices
-#  - URL for additional packages repository
+#  - <image-server-address> - IP or hostname with port if different than 80 used for connecting to the local web server
+#  - <image-name> - a name for the image. allows using different ostree image for different types of edge devices
+#  - <agent-server-url> - URL for connecting to the server that monitors edge devices
+#  - [packages-repo-url] - optional, URL for additional packages repository
 # script assumes that files `blueprint.template` and `ks.cfg.template` are present in the working directory
+
+function cleanup() {
+    rm -f temp.out 2 &> /dev/null
```

Comment:
I don't think you need the redirection to /dev/null when u use '-f'

Reply:
correct, added it to ignore a specific case, I'll change this function to:
```
function cleanup() {
    rm -f temp.out
    rm -f $REPO_DIR
    if [ -n "$BUILD_ID" ]; then
        rm -f $BUILD_ID-commit.tar
    fi
}
```

since without it, if trap happens before the $BUILD_ID was defined, then the command fails (interpreted to `rm -f -commit.tar` that fails)